src/renderer/store/modules/peer.js

```diff
@@ -100,7 +100,7 @@ export default {
      */
     best: (_, getters) => (ignoreCurrent = true, networkId = null) => {
       const peers = getters['bestPeers'](undefined, ignoreCurrent)
-      if (!peers || !peers.length) {
```

Comment:
why was this removed? 🤔 i guess it still works as intended, but the modified condition does not stop the execution as intended.